outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -69,8 +70,7 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
         }
 
         this.setState({isLoading: true});
-        this.enriched = true;
-        const enrichRequest = sendHttpRequest(HttpVerb.POST, api.baseURL + api.enrichCompany,
+        const enrichRequest = sendHttpRequest(HttpVerb.POST, api.baseURL + api.enrichAndCreateCompany,
```

Comment:
Why not rename "enrichCompanyForPartnerRequest" to "enrichAndCreate" and the new one to "enrichAndUpdate"?

That way you keep similar names to the controller endpoints (and the names are more meaningful).

Reply:
Yes, thanks for the hint, these names are much shorter!